Makefile

```diff
@@ -123,15 +123,15 @@ endif
-	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_2_2_COINBASE) --port $(ZONE_2_2_PORT_TCP) --http.port $(ZONE_2_2_PORT_HTTP) --ws.port $(ZONE_2_2_PORT_WS) --dom.url $(ZONE_2_2_DOM_URL):$(REGION_2_PORT_WS)                                 --region 2 --zone 2 >> nodelogs/zone-2-2.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_0_0_COINBASE) --port $(ZONE_0_0_PORT_TCP) --http.port $(ZONE_0_0_PORT_HTTP) --ws.port $(ZONE_0_0_PORT_WS) --dom.url $(ZONE_0_0_DOM_URL):$(REGION_0_PORT_WS) $(if $(METRICS),--metrics) --region 0 --zone 0 >> nodelogs/zone-0-0.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_0_1_COINBASE) --port $(ZONE_0_1_PORT_TCP) --http.port $(ZONE_0_1_PORT_HTTP) --ws.port $(ZONE_0_1_PORT_WS) --dom.url $(ZONE_0_1_DOM_URL):$(REGION_0_PORT_WS) $(if $(METRICS),--metrics) --region 0 --zone 1 >> nodelogs/zone-0-1.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_0_2_COINBASE) --port $(ZONE_0_2_PORT_TCP) --http.port $(ZONE_0_2_PORT_HTTP) --ws.port $(ZONE_0_2_PORT_WS) --dom.url $(ZONE_0_2_DOM_URL):$(REGION_0_PORT_WS) $(if $(METRICS),--metrics) --region 0 --zone 2 >> nodelogs/zone-0-2.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_1_0_COINBASE) --port $(ZONE_1_0_PORT_TCP) --http.port $(ZONE_1_0_PORT_HTTP) --ws.port $(ZONE_1_0_PORT_WS) --dom.url $(ZONE_1_0_DOM_URL):$(REGION_1_PORT_WS) $(if $(METRICS),--metrics) --region 1 --zone 0 >> nodelogs/zone-1-0.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_1_1_COINBASE) --port $(ZONE_1_1_PORT_TCP) --http.port $(ZONE_1_1_PORT_HTTP) --ws.port $(ZONE_1_1_PORT_WS) --dom.url $(ZONE_1_1_DOM_URL):$(REGION_1_PORT_WS) $(if $(METRICS),--metrics) --region 1 --zone 1 >> nodelogs/zone-1-1.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_1_2_COINBASE) --port $(ZONE_1_2_PORT_TCP) --http.port $(ZONE_1_2_PORT_HTTP) --ws.port $(ZONE_1_2_PORT_WS) --dom.url $(ZONE_1_2_DOM_URL):$(REGION_1_PORT_WS) $(if $(METRICS),--metrics) --region 1 --zone 2 >> nodelogs/zone-1-2.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_2_0_COINBASE) --port $(ZONE_2_0_PORT_TCP) --http.port $(ZONE_2_0_PORT_HTTP) --ws.port $(ZONE_2_0_PORT_WS) --dom.url $(ZONE_2_0_DOM_URL):$(REGION_2_PORT_WS) $(if $(METRICS),--metrics) --region 2 --zone 0 >> nodelogs/zone-2-0.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_2_1_COINBASE) --port $(ZONE_2_1_PORT_TCP) --http.port $(ZONE_2_1_PORT_HTTP) --ws.port $(ZONE_2_1_PORT_WS) --dom.url $(ZONE_2_1_DOM_URL):$(REGION_2_PORT_WS) $(if $(METRICS),--metrics) --region 2 --zone 1 >> nodelogs/zone-2-1.log 2>&1 &
+	@nohup $(BASE_CMD) --miner.etherbase $(ZONE_2_2_COINBASE) --port $(ZONE_2_2_PORT_TCP) --http.port $(ZONE_2_2_PORT_HTTP) --ws.port $(ZONE_2_2_PORT_WS) --dom.url $(ZONE_2_2_DOM_URL):$(REGION_2_PORT_WS) $(if $(METRICS),--metrics) --region 2 --zone 2 >> nodelogs/zone-2-2.log 2>&1 &
```

Comment:
My concern was that it would also apply to region and prime which aren't doing as much processing as zone. But the metrics seem reasonable with this as well

Reply:
oops, I didn't notice that. Since it should only be enabled in the zones, and not prime and region, then I think what you had is correct